.github/CHANGELOG.md

```diff
@@ -80,6 +80,8 @@
 
 <h3>Improvements</h3>
 
+* The Hamiltonian class was moved to the `ops/qubit` folder, since it is now an observable.
```

Comment:
```suggestion
* The Hamiltonian class was moved to the `ops/qubit` folder, since it is now an observable.
  [(#1534)](https://github.com/PennyLaneAI/pennylane/pull/1534)
```